parlai/zoo/tod/tod_base_yes_api.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Task-Oriented Dialog pretrained model *without* usage of API Schemas.
```

Comment:
we should probably be more specific. your paper will not be the only tod paper.

Reply:
Good catch